frontend/lib/src/helper/snackbar/show_snackbar.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+
+/*
+  Reusable snackbar dialog
```

Comment:
```suggestion
  Reusable snackbar dialog?
```

does it have dialog functionality? user can click cancel or confirm button?